spec/collections_publisher/archiving_child_topic_spec.rb

```diff
@@ -0,0 +1,70 @@
+feature "Archiving a child topic on Collections Publisher", collections_publisher: true do
+  include CollectionsPublisherHelpers
+
+  let(:topic_title) { title_with_timestamp }
+  let(:topic_slug) { slug_with_timestamp }
+
+  let(:child_slug) { slug_with_timestamp }
+  let(:child_title) { title_with_timestamp }
+  let(:link) { ["/topic", topic_slug, child_slug].join('/') }
+
+  scenario "Archiving a child topic" do
+    given_there_is_a_published_child_topic
+    when_i_archive_it
+    then_i_am_redirected_on_gov_uk
```

Comment:
should we be more specific about the redirect e.g. "then_i_am_redirected_to_the_parent_on_gov_uk" ?